docker-compose.yml

```diff
@@ -10,11 +10,20 @@ services:
     ports:
       - "8000:8000"
     restart: unless-stopped
+    environment:
+      INDEXDB: "true"
+      METADATA: "/data/mmd_xml/nbs"
+      OUTPUT_ISO: "/data/iso_xml/nbs"
     volumes:
       - ./Volumes/CSW/conf/pycsw.cfg:/etc/pycsw/pycsw.cfg
       - ./Volumes/CSW/commands/pycsw_setup.sh:/usr/local/bin/pycsw_setup.sh
+      - ./Volumes/CSW/commands/mmd2isofix.py:/usr/local/bin/mmd2isofix.py
+      - ./Volumes/CSW/commands/mmd-to-iso.xsl:/usr/local/share/mmd-to-iso.xsl
       - ./Volumes/data:/data
-    command: /bin/sh /usr/local/bin/pycsw_setup.sh
+    # command: /bin/sh /usr/local/bin/pycsw_setup.sh
+    entrypoint:
```

Comment:
Developer of pycsw both prepare database and start service in one file. So this is OK for us to do.